integration/integration_api_test.go

```diff
@@ -0,0 +1,58 @@
+// +build integration
+// can be execute with go test -tags=integration ./integration/...
+
+package main
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/piperenv"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPiperGithubPublishRelease(t *testing.T) {
+	t.Parallel()
+	token := os.Getenv("PIPER_INTEGRATION_GITHUB_TOKEN")
+	if len(token) == 0 {
+		t.Fatal("No GitHub token maintained")
+	}
+
+	owner := os.Getenv("PIPER_INTEGRATION_GITHUB_OWNER")
+	if len(owner) == 0 {
+		owner = "OliverNocon"
```

Comment:
Is it intended to have a user here hard coded (also as a fallback)?

Reply:
This is a sample project to verify Github Release creation: https://github.com/OliverNocon/piper-integration/releases